src/mlpack/core/util/binding_details.hpp

```diff
@@ -0,0 +1,44 @@
+/**
+ * @file core/util/binding_details.hpp
+ * @author Yashwant Singh Parihar
+ *
+ * This defines the structure that holds documentation details for bindings.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_UTIL_BINDING_DETAILS_HPP
+#define MLPACK_CORE_UTIL_BINDING_DETAILS_HPP
+
+#include <mlpack/prereqs.hpp>
+#include "program_doc.hpp"
+
+namespace mlpack {
+namespace util {
+
+/**
+ * This structure holds all of the information about bindings documentation.
+ */
+struct BindingDetails
+{
+  //! Name of the default module.
+  std::string programName;
+  //! A short two-sentence description of the program, what it does, and what
```

Comment:
```suggestion
  //! A short two-sentence description of the binding, what it does, and what
```
Just a little pedantry since we are making bindings for any language now, not just for command-line programs. :+1:

Reply:
Done.